<?xml version="1.0"?>
<!-- vim: set fenc=utf-8 et sts=4 ft=xml : -->
<launch>
    <arg name="bagfile"/>

    <param name="use_sim_time" value="true" type="bool"/>

    <node
        name="replay"
        pkg="rosbag"
        type="play"
        output="screen"
        args="--pause --clock
            --bags $(arg bagfile)
            --topics
            /tf
            /motion/platform_joint_state
            /motion/odometry
            "
            >
            <remap from="/motion/odometry" to="ekf_odometry"/>
    </node>

    <node
        name="odometry_node"
        pkg="platform_motion"
        type="lls_odometry"
        output="screen"
        >
        <!--
        launch-prefix="tmux_new_window gdb - -args "
        -->
        <remap from="platform_joint_state" to="/motion/platform_joint_state"/>
        <remap from="odometry" to="lls_odometry"/>
    </node>

</launch>
